man/man1/exiv2.1

```diff
@@ -39,31 +39,35 @@ lB lB lB lB lB lB
 _	_	_	_	_	_
 l l l l l l.
 Type	Exif	IPTC	XMP	Image Comments	ICC Profile
-JPEG	Read/Write	Read/Write	Read/Write	Read/Write	Read/Write
-EXV	Read/Write	Read/Write	Read/Write	Read/Write Read/Write
+ARW	Read	Read	Read	-	-
+AVIF	Read	Read	Read	-	-
+BMP	-	-	-	-	-
 CR2	Read/Write	Read/Write	Read/Write	-	Read/Write
+CR3	Read	Read	Read	-	Read
 CRW	Read/Write	-	-	Read/Write	-
-MRW	Read	Read	Read	-	-
-TIFF	Read/Write	Read/Write	Read/Write	-	Read/Write
-WEBP	Read/Write	-	Read/Write	-	Read/Write
 DNG	Read/Write	Read/Write	Read/Write	-	Read/Write
+EPS	-	-	Read/Write	-	-
+EXV	Read/Write	Read/Write	Read/Write	Read/Write Read/Write
+GIF	-	-	-	-	-
+HEIC	Read	Read	Read	-	-
+HIF	Read	Read	Read	-	-
```

Comment:
Does "Type" mean file extension here? If so, also need to add HEIF, otherwise, change HIF->HEIF.